packages/js/components/src/experimental-select-control/select-control.tsx

```diff
@@ -297,7 +303,10 @@ function SelectControl< ItemType = DefaultItemType >( {
 					...getDropdownProps( {
 						preventKeyAction: isOpen,
 					} ),
-					className: 'woocommerce-experimental-select-control__input',
+					className: classnames(
+						'woocommerce-experimental-select-control__input',
+						listItemClassName
```

Comment:
This will add `woocommerce-experimental-select-control__list-item` to the input which seems semantically incorrect and could lead to styling errors.

Reply:
Yeah, I used that name because at the end we see a list of inputs. But I'm totally ok removing that class. I did that in commit [57f2879](https://github.com/woocommerce/woocommerce/pull/39574/commits/57f2879ccc1ee05165f56e775a8c6ca0fce5b911)